src/Setup/Installer/tools/uninstall.ps1

```diff
@@ -24,13 +19,14 @@ try {
     $vsDir = $vsInstance.installationPath.Trim("\")
     $vsId = $vsInstance.instanceId
     $vsMajorVersion = $vsInstance.installationVersion.Split(".")[0]
+    $mefCacheFolder = Get-MefCacheDir -vsMajorVersion $vsMajorVersion -vsId $vsId -rootSuffix $rootSuffix
+    
+    Stop-Processes $vsDir (Split-Path $mefCacheFolder)
```

Comment:
Why not rename `Get-MefCacheDir` to just return the hive dir, and then have the MEF cache thing `Join-Path` the extra subfolder? Otherwise it's basically sticking an extra path on only for you to strip it back off...